src/IJulia.jl

```diff
@@ -584,4 +572,92 @@ include("inline.jl")
 include("kernel.jl")
 include("precompile.jl")
 
+#######################################################################
+# App entry point for `ijulia` command
+
+# Mutable function references for testing - allows tests to mock these functions
+notebook_cmd::Function = notebook
+jupyterlab_cmd::Function = jupyterlab
```

Comment:
Is it possible to pass these in as arguments to `@main`? I slightly prefer that over global variables.